src/components/chart/annotations-options.js

```diff
@@ -0,0 +1,76 @@
+import ChartConstants from './chart-constants';
+
+class AnnotationsOptions {
+    constructor(annotations) {
+        this.constants = ChartConstants.constants();
+        this.annotations = annotations;
+    }
+
+    getAnnotationsOptionsDesktop = () => {
+        let annotations = [
+            {
+                labels: [],
+                labelOptions: {
+                    shape: 'connector',
+                    borderColor: this.constants.labelColor,
+                    padding: 3,
+                    style: {
+                        color: this.constants.labelColor,
+                        fontSize: this.constants.desktopFontSize,
```

Comment:
This is currently displaying as 14px for me at desktop - based on the ACs it should be 16px

Reply:
Thanks @janehughes - this has made me realise we have smaller fonts than the ACs specify throughout the charts, not just here. But changing them to larger looks quite large, and increases the chance of overlapping text. I'll raise this as a question on slack, but hopefully shouldn't block this PR, as we can do it as a separate fix for everything if needed.

Reply:
I've chatted with Erin from datavis about this - the AC is my mistake. We do need a ticket to adjust the mobile font-sizes, but I'll cover that separately. I'm going to do a sweep of tickets and update the ACs regarding font size this afternoon.